website/templates/emails/discussions_email_rejected.txt.mako

```diff
@@ -0,0 +1,19 @@
+Hello${' {}'.format(user.fullname) if user else ''},
+
+The email that you sent to ${target_address} was rejected, and was therefore not forwarded.
+
+%if reason == 'no_user':
```

Comment:
Is it possible to use the [constants](https://github.com/CenterForOpenScience/osf.io/blob/9e26f9dc5f535db5d760b7d7038da84ca3960122/website/project/model.py#L4621-L4626) here, since we have them?